@here/harp-examples/src/getting-started_orbiting-view.ts

```diff
@@ -60,7 +60,8 @@ export namespace CameraOrbitExample {
         const mapView = new MapView({
             canvas,
             projection: sphereProjection,
-            theme: "resources/berlin_tilezen_base_globe.json"
+            theme: "resources/berlin_tilezen_base_globe.json",
+            watermark: URL.createObjectURL(new Blob([watermarkLogo], { type: "image/svg+xml" }))
```

Comment:
This `URL.createObjectURL(new Blob(...))` really begs to be moved to common

Reply:
Unfortunately it is impossible to use `Blob` in `config.ts` because this file is imported in `scripts/credentials.ts` which is run in Node.JS environment and has no `Blob`. Have to apply some hacks for this.